package.json

```diff
@@ -36,7 +36,6 @@
     "@aws-cdk/aws-lambda": "1.71.0",
     "@aws-cdk/aws-logs": "1.71.0",
     "@aws-cdk/aws-logs-destinations": "1.71.0",
-    "@aws-cdk/core": "1.71.0",
-    "crypto": "^1.0.1"
```

Comment:
Was it intentional to remove `crypto` here? If so, I would also expect the `yarn.lock` file to change.

Reply:
@nhinsch Every time I installed the dependencies it would say crypto is depreciated from NPM so I figured it was not needed anymore.

Reply:
It is now removed from the yarn lock